truss/test_data/truss_container_fs/app/common/truss_server.py

```diff
@@ -0,0 +1,195 @@
+import json  # noqa: E402
+import logging  # noqa: E402
+from http import HTTPStatus  # noqa: E402
+
+import numpy as np  # noqa: E402
+import tornado.web  # noqa: E402
+from common.lib_support import ensure_kfserving_installed
+from common.serialization import DeepNumpyEncoder  # noqa: E402
+from common.serialization import truss_msgpack_deserialize  # noqa: E402
+from common.serialization import truss_msgpack_serialize  # noqa: E402
+from common.util import (  # noqa: E402
+    assign_request_to_inputs_instances_after_validation,
+)
+from kfserving.handlers.http import HTTPHandler  # noqa: E402
+from kfserving.kfserver import LivenessHandler  # noqa: E402; noqa: E402
+from kfserving.kfserver import HealthHandler, KFServer, ListHandler
+from pythonjsonlogger import jsonlogger  # noqa: E402
+
+ensure_kfserving_installed()
+
+
+def _configure_logging():
```

Comment:
Can you help me understand why we have these same handlers copies over that are part of b10server.py? I thought we would have our patch endpoint, but that would be it on the pynode. Everythign else could stay the same. i.e. we only added the control plan server onto the pynode

Reply:
So pynodes are not involved at all in this change, this change is only focused on Truss, which relates to models. b10server(in the other repo) is stale code I think, no longer used. Let me sync with you about this offline.